app/templates/views/styleguide.html

```diff
@@ -69,141 +60,333 @@ <h2 class="heading-large">Browse list</h2>
+  <fieldset>
+    <legend class="sr-only">Select templates and folders</legend>
+
+    <div
+      class="template-list-item template-list-item-with-checkbox template-list-item-without-ancestors"
+      data-template-category="info-request"
+      data-notification-type="email"
+    >
+      <div class="multiple-choice mt-4">
+        <input
+          id="templates-or-folder-aee9478d-bfee-49de-8d9b-f21894ddbf21"
+          name="templates_and_folders"
+          type="checkbox"
+          value="aee9478d-bfee-49de-8d9b-f21894ddbf21"
+        />
+
+        <label for="templates-or-folder-aee9478d-bfee-49de-8d9b-f21894ddbf21">
+          <span class="sr-only">Select the Lang tag tests Email template</span>
+        </label>
+      </div>
+
+      <p class="message-name2">
+        <a
+          href="/services/1c472757-0c28-4b5b-b58b-ac31bca6ac51/templates/aee9478d-bfee-49de-8d9b-f21894ddbf21"
+          aria-describedby="sr-aee9478d-bfee-49de-8d9b-f21894ddbf21"
+          class="block sm:flex py-4 items-center no-underline"
+        >
+          <span class="live-search-relevant block flex-grow underline"
+            >Lang tag tests</span
+          >
+          <span
+            class="text-small no-underline block w-full sm:w-1/4 font-normal"
+            >Email template</span
+          >
+          <span
+            class="text-small no-underline block w-full sm:w-1/4 font-normal"
+            >Information request</span
+          >
+        </a>
       </p>
     </div>
-  </div>
-
-  <h2 class="heading-large">Textbox</h2>
-  {{ textbox(form.username) }}
-  {{ textbox(form.password) }}
-  {{ textbox(form.message, highlight_tags=True) }}
-  {{ textbox(form.code, width='w-full md:w-1/4') }}
-
-  <h2 class="heading-large">File upload</h2>
-  {{ file_upload(form.file_upload) }}
-
-  <h2 class="heading-large">API key</h2>
 
-  {{ api_key('d30512af92e1386d63b90e5973b49a10') }}
-
-
-  <h2 class="heading-large">Formatted list</h2>
-
-  <p>
-    {{ 'A' | formatted_list(prefix="one item called") }}
-  </p>
+    <div
+      class="template-list-item template-list-item-with-checkbox template-list-item-without-ancestors"
+      data-template-category="info-request"
+      data-notification-type="sms"
+    >
+      <div class="multiple-choice mt-4">
+        <input
+          id="templates-or-folder-4bee849e-19e0-4ab4-947a-8b3585ec6546"
+          name="templates_and_folders"
+          type="checkbox"
+          value="4bee849e-19e0-4ab4-947a-8b3585ec6546"
+        />
+
+        <label for="templates-or-folder-4bee849e-19e0-4ab4-947a-8b3585ec6546">
+          <span class="sr-only">Select the Lexical test Email template</span>
+        </label>
+      </div>
+
+      <p class="message-name2">
+        <a
+          href="/services/1c472757-0c28-4b5b-b58b-ac31bca6ac51/templates/4bee849e-19e0-4ab4-947a-8b3585ec6546"
+          aria-describedby="sr-4bee849e-19e0-4ab4-947a-8b3585ec6546"
+          class="block sm:flex py-4 items-center no-underline"
```

Comment:
```suggestion
          class="block sm:flex py-4 items-baseline no-underline"
```

Like this, but we can tackle it when we're updating the real component